src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationOverlay.tsx

```diff
@@ -0,0 +1,296 @@
+  [provider: string]: Cluster[]
+}
+
+enum ProviderSelectMessage {
+  SingleProvider = 'This account uses the following provider and region to store time series data for this organization.',
+  MultiProvider = 'Tell us where you would like to store the time series data for this organization',
+}
+
+// Selectors
+import {
+  selectCurrentAccountId,
+  selectQuartzOrgsContents,
+  selectQuartzOrgsStatus,
+} from 'src/identity/selectors'
+
+// Notifications
+import {notify} from 'src/shared/actions/notifications'
+import {orgCreateSuccess} from 'src/shared/copy/notifications'
+
+// Thunks
+import {getQuartzOrganizationsThunk} from 'src/identity/quartzOrganizations/actions/thunks'
+
+// Utils
+import {generateProviderMap} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/utils/generateProviderMap'
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+
+export const CreateOrganizationOverlay: FC = () => {
+  const dispatch = useDispatch()
+  const {onClose} = useContext(OverlayContext)
+
+  // Selectors
+  const currentAccountId = useSelector(selectCurrentAccountId)
+  const organizations = useSelector(selectQuartzOrgsContents)
+  const orgsLoadedStatus = useSelector(selectQuartzOrgsStatus)
+
+  // Local State
+  // Dropdown State
+  const [providerMap, setProviderMap] = useState({})
+  const [currentProvider, setCurrentProvider] = useState(null)
+  const [currentRegion, setCurrentRegion] = useState(null)
+  const [newOrgName, setNewOrgName] = useState('')
+  const providerNames = Object.keys(providerMap)
+  const numProviders = providerNames.length
+  const providerHelpText =
+    numProviders > 1
+      ? ProviderSelectMessage.MultiProvider
+      : ProviderSelectMessage.SingleProvider
+
+  // Error Handling State
+  const [createOrgButtonStatus, setCreateOrgButtonStatus] = useState(
+    ComponentStatus.Disabled
+  )
+  const [networkErrorMsg, setNetworkErrorMsg] = useState(
+    OrgOverlayNetworkError.None
+  )
+  const [validationMsg, setValidationMsg] = useState(
+    OrgOverlayValidationError.None
+  )
+
+  // Spinner State
+  const loadingComplete = Boolean(currentProvider) && Boolean(currentRegion)
+  const loadingState = loadingComplete
+    ? RemoteDataState.Done
+    : RemoteDataState.Loading
+
+  // Event Handlers
+  const handleCreateOrg = async () => {
+    try {
+      setCreateOrgButtonStatus(ComponentStatus.Loading)
+      await createNewOrg({
+        orgName: newOrgName,
+        provider: currentProvider,
+        region: currentRegion,
+      })
+      setCreateOrgButtonStatus(ComponentStatus.Default)
+      onClose()
+      dispatch(notify(orgCreateSuccess()))
+      // Will make another API call to the allowance endpoint in PR resolving this issue.
+      // https://github.com/influxdata/ui/issues/6267
+      // Need a parallel update to global header state to ensure list of orgs remains updated.
+      dispatch(getQuartzOrganizationsThunk(currentAccountId))
+    } catch (err) {
```

Comment:
I hear you, but this falls into the same category as `evt` versus `event` in my view. I'd like to avoid calling it `error` because a Control+F (esp. if not case sensitive) will pull up actual references to the `Error` constructor as opposed to errors in try/catch blocks.